docs/synthetics/user-journey-scripts.md

```diff
@@ -107,3 +107,30 @@ The logs will also contain the Sematext API response indicating the sync failure
+
+### Using Secrets
+To showcase an example of where Secrets could be used, we'll use the User Journey script in the screenshot below.
+
+ ![Browser monitor script with plaintext credentials](../images/synthetics/secrets-script-plaintext.png)
+
+As you can see, this User Journey script navigates to some page, fills in input fields with a username and a password, logs in and takes a screenshot. However, note that the credentials in this example are in their plaintext form. You can avoid having plaintext credentials stored in your scripts and instead substitute them with placeholders, while the actual credentials themselves are securely stored in the AWS Secrets Manager. To do this, you can click on the **Add Sensitive Data** button located right above the script editor. Upon doing so, you'll see the menu for adding sensitive data appear on the right side.
+
+ ![Adding sensitive data for Browser monitor scripts ](../images/synthetics/secrets-script-creating-secret.png)
+
+Here we can assign placeholder values which will be displayed instead of the actual credentials. Instead of our actual username, we'll use `$__username__` and we'll replace the password with `$__password__`. Note that these placeholders (or *Secret Keys*) must be in this format:  
+- `$__exampleSecretKey__`
+
+meaning that they must start with `$__` and end with `__`, and they may only contain alphanumeric characters and underscores. Once you're satisfied with the values you've defined, click the Save button in the menu's bottom right corner. You'll then see the keyboard shortcut tooltip below the script editor, which will let you quickly insert the placeholders you've defined into your script, as seen in the screenshot below.
+
+ ![Browser monitor script secrets autofill feature ](../images/synthetics/secrets-script-autofill.png)
+
+If we select the `$__password__` placeholder from the tooltip, it will be inserted at the cursor's position, as seen in the image below.
+
+ ![Browser monitor script with secrets ](../images/synthetics/secrets-script-with-secrets.png)
+
+Once you finish adding your sensitive data, keep in mind that the secret values you've filled in won't be accessible on Sematext Cloud anymore. They will only be used while executing your script, allowing you to define the credentials once and avoid having to worry about who has access to them. Because we can't access these secrets from Sematext Cloud, that will also mean that you'll have to input all your secret values if you end up having to edit them. The only values we'll be able to display are the placeholders you've defined, so that you can conveniently edit your User Journey scripts and easily reference your credentials without having to edit them. An example of what the **Add Sensitive Data** menu will look like upon saving and then editing the monitor is shown in the screenshot below.
```

Comment:
How about we simplify `if you end up having to edit them` to `if you want to edit them`

Reply:
Done 👍